src/fr-webauthn/helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { ParsedCredential } from './interfaces';
+
+function parsePubKeyArray(s: string | unknown[]): PublicKeyCredentialParameters[] | undefined {
```

Comment:
I think it's consistent with something like `parseInt`, no?  I'm actually a fan of one-letter variables in small pure functions like this because it helps avoid line-wrapping, and the functions are so small you're not at risk of forgetting what `s` is.  Another great topic for the style guide!